Language/Functions/External Interrupts/attachInterrupt.adoc

```diff
@@ -147,5 +147,8 @@ For Uno WiFiRev.2, Due, Zero, MKR Family and 101 boards the *interrupt number =
 [float]
 === See also
 
+[role="language"]
+* #LANGUAGE# link:../../external-interrupts/detachinterrupt[detachInterrupts()]
 --
+
```

Comment:
```suggestion

--
```
Use formatting style established by the reference sample:

Let's leave this part as it was because it is unrelated to the proposal and this is how the reference sample is formatted:

https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc